Yosemite/Yosemite/Model/Enums/CardPresentPaymentsOnboardingState.swift

```diff
@@ -131,4 +131,13 @@ extension CardPresentPaymentOnboardingState {
             return false
         }
     }
+
+    public var isCountryNotSupported: Bool {
+        if case .countryNotSupported(_) = self {
+            return true
+        } else {
+            return false
+        }
+
```

Comment:
Nit: Extra space :D

Reply:
NIce catch! Removed in 06d4da432ffd30dd4c16d5474dbd8b793be4605c